app/controllers/api/v1/spots_controller.rb

```diff
@@ -9,6 +9,10 @@ def index
         @spots = current_user.spots
       end
 
+      def destroy
+        current_user.spots.destroy(params[:id])
```

Comment:
Use bang (!) symbol, this way you will throw the exceptions if the destroy method fail for some reason.

`current_user.spots.destroy!(params[:id])`